src/account-link.ts

```diff
@@ -0,0 +1,6 @@
+import prefixForNetwork from './prefix-for-network'
+
+export = function (address: string, network: string): string {
```

Comment:
this wasn't introduced here, but it would be a nice thing to do to add names to these functions for debugability and for IntelliSense

Reply:
Good idea!  Updated!